agent/hcp/client.go

```diff
@@ -18,6 +18,7 @@ import (
 
 	"github.com/hashicorp/consul/agent/hcp/config"
 	"github.com/hashicorp/consul/version"
+	consulversion "github.com/hashicorp/consul/version"
```

Comment:
This is the same dep imported on the line above.

Reply:
good find, fixed.